### DISCUSSION TITLE
Make checklists usable to viewers

### DISCUSSION THREAD

**Discussion by jayb323** [outline/outline#3406] [Ideas]
Hey there,

It would be very useful if checklists could be usable (but not saved) by viewers. 

The reason is that if the document is being used for a procedure, it's almost mandatory that it should have a checklist for the user to use. Usually, procedure software would allow the user to check the list but not change anything.

I'm not sure if that's hard to do (it doesn't seem it), and I can't think of any way this could affect the use of checklists generally other than enhancing their use.

Cheers,

Jay

**Comment by tommoor** [comment]
This seems like it would be quite confusing – nowhere else are changes unsaved. The recommended way right now to do this would be to create a template with your checklist and then duplicate it to fill out. Is it not also important to have a record of the procedure being run?

**Reply by jayb323** [reply]
The issue with that is there might be hundreds of procedures. I suppose there could be a template to match the procedures but then they need to follow the procedure and open a separate checklist template. Having the checklist in the procedure forces them to open the procedure to use the checklist (which is really a mini procedure), so it has other benefits.

It's not important to have a record of it being run in the procedure document, in my view. That gets more into workflows and even process orchestration rather than documentation.

Other systems (like Confluence, for example), allow viewers to check/uncheck checklists in the viewer users own session only. In fact, you can't make Confluence not do that.

**Reply by jayb323** [reply]
Just did a fact check, it appears you can make Confluence **not** do that now (i.e. make it only checkable by people with permissions).
